src/Components/Home.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import './Welcome.css'
+import gql from 'graphql-tag'
+import { CachePersistor } from 'apollo-cache-persist'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { useQuery } from '@apollo/react-hooks'
+import { useHistory } from 'react-router-dom'
+import * as Storage from './Storage'
+
+const cache = new InMemoryCache()
+const persistor = new CachePersistor({ cache, storage: window.localStorage })
```

Comment:
Don't create new instances of the Caché. Always use the same instance that you created for the Apollo Client. You could even use the Apollo Client instance to clear the caché: https://www.apollographql.com/docs/react/api/apollo-client/#ApolloClient.clearStore